apps/main/src/loan/components/PageLoanCreate/Page.tsx

```diff
@@ -202,13 +194,15 @@ const Page = () => {
           )}
           <Stack>
             <MarketDetails {...marketDetails} />
-            <MarketInformationComp
-              llamma={llamma}
-              llammaId={llammaId}
-              chainId={rChainId}
-              chartExpanded={chartExpanded}
-              page="create"
-            />
+            {market && (
```

Comment:
Why is this added? It results in the page jumping during load

Reply:
For some reason the old code type asserted that llamma couldn't be `undefined`. But I'm not sure if that assertion is valid. How can it be asserted to not be null when stuff is being loaded? Either the assertion is somehow true and this won't jump, or stuff was being shown prematurely in the old verison. I'd rather keep this check to make sure there's no errors. If people never complained about the page jumping it's probably a safe assumption?

Reply:
People dont complain because it works, it just looks unprofessional